releasenotes/notes/2.0/fixes_13858-2dacfc0431c1a6ea.yaml

```
@@ -1,10 +1,20 @@
 ---
 upgrade_synthesis:
   - |
-    The ``atomic_evolution`` argument to :class:`.ProductFormula` (and its
-    subclasses :class:`.QDrift`, :class:`.LieTrotter` and :class:`SuzukiTrotter` ) 
-    has a new function signature. Rather than taking some Pauli
-    operator and time coefficient and returning the evolution circuit, the new
-    function takes in an existing circuit and should append the evolution of the
-    provided Pauli and given time to this circuit. This new implementation
-    benefits from significantly better performance.
+    The ``atomic_evolution`` callable argument of :class:`.ProductFormula` (and its
+    subclasses :class:`.QDrift`, :class:`.LieTrotter`, and :class:`SuzukiTrotter` ) 
+    has a new function signature. The old signature would take some Pauli
+    operator and time coefficient and return the evolution circuit::
+
+      def atomic_evolution(pauli_op: SparsePauliOp, time: float):
```

If the function already has partial typehints...
```suggestion
      def atomic_evolution(pauli_op: SparsePauliOp, time: float) -> QuantumCircuit:
```